tests/Unit/Crypt/RSA/CreateKeyTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @author    Jim Wigginton <terrafrost@php.net>
+ * @copyright 2013 Jim Wigginton
+ * @license   http://www.opensource.org/licenses/mit-license.html  MIT License
+ */
+
+use phpseclib\Crypt\RSA;
+
+class Unit_Crypt_RSA_CreateKeyTest extends PhpseclibTestCase
+{
+    static protected $privatekey;
+    static protected $publickey;
+
+    public function testCreateKey()
+    {
+        extract(RSA::createKey(512));
+        $this->assertInstanceOf('\phpseclib\Crypt\RSA', $privatekey);
+        $this->assertInstanceOf('\phpseclib\Crypt\RSA', $publickey);
+        $this->assertNotEmpty("$privatekey");
+        $this->assertNotEmpty("$publickey");
+
+        self::$publickey = $publickey;
```

Comment:
I'm not sure what you mean. Do you mean having `testEncryptDecrypt` use a dataprovider that generates the keys?